corehq/apps/data_analytics/tasks.py

```diff
@@ -0,0 +1,36 @@
+from celery.schedules import crontab
+from celery.task import periodic_task
+from celery.utils.log import get_task_logger
+import datetime
+
+from corehq.apps.data_analytics.malt_generator import MALTTableGenerator
+from dimagi.utils.dates import DateSpan
+from dimagi.utils.django.email import send_HTML_email
+
+
+logger = get_task_logger(__name__)
```

Comment:
if it's a lot of errors we may not want to email on each one - and if you want the emails then `soft_assert` just to yourself would be more appropriate.

if you just want the logs available then figuring out the file logging seems better than using email